pyproject.toml

```diff
@@ -11,10 +11,11 @@ classifiers = [
   'Operating System :: MacOS',
   'Operating System :: Microsoft :: Windows',
   'Operating System :: POSIX',
+  'Programming Language :: Python :: 3.9',
   'Programming Language :: Python :: 3.10',
   'Programming Language :: Python :: 3.11',
   'Programming Language :: Python :: 3.12',
-  'Programming Language :: Python :: 3.9',
+  'Programming Language :: Python :: 3.13',
```

Comment:
We need #6879 and we also need to add testing for it.  Either way this is a separate PR.  See https://github.com/pyvista/pyvista/pull/6879#discussion_r1975992219